config/tests/jobs/presubmit-jobs.yaml

```diff
@@ -0,0 +1,158 @@
+    - name: pull-kubernetes-e2e-gce-storage-slow
+      optional: true
+      run_if_changed: ^(pkg\/controller\/volume|pkg\/kubelet\/volumemanager|pkg\/volume|pkg\/util\/mount|test\/e2e\/storage|test\/e2e\/testing-manifests\/storage-csi)
+    - name: pull-kubernetes-e2e-gce-storage-snapshot
+      optional: true
+      run_if_changed: ^(pkg\/controller\/volume|pkg\/kubelet\/volumemanager|pkg\/volume|pkg\/util\/mount|test\/e2e\/storage|test\/e2e\/testing-manifests\/storage-csi)
+    - name: pull-kubernetes-e2e-gci-gce-autoscaling
+      optional: true
+      run_if_changed: ^(cluster/gce/manifests/cluster-autoscaler.manifest$|cluster/addons/rbac/cluster-autoscaler)
+    - name: pull-kubernetes-e2e-gci-gce-ingress
+      optional: true
+      run_if_changed: ^(test/e2e/network/|pkg/apis/networking)
+    - name: pull-kubernetes-e2e-gci-gce-ipvs
+      optional: true
+      run_if_changed: ^(test/e2e/network/|pkg/apis/networking|pkg/.*/ipvs/)
+    - name: pull-kubernetes-e2e-inplace-pod-resize-containerd-main-v2
+      optional: true
+      run_if_changed: test/e2e/node/pod_resize.go|pkg/kubelet/kubelet.go|pkg/kubelet/kubelet_pods.go|pkg/kubelet/kuberuntime/kuberuntime_manager.go
+    - name: pull-kubernetes-e2e-kind-alpha-beta-features
+      optional: true
+      run_if_changed: ^pkg/features/
+    - name: pull-kubernetes-e2e-kind-ipvs
+      optional: true
+      run_if_changed: ^(test/e2e/network/|pkg/apis/networking|pkg/proxy/ipvs/)
+    - name: pull-kubernetes-e2e-kind-kms
+      optional: true
+      run_if_changed: staging/src/k8s.io/apiserver/pkg/storage/value/|staging/src/k8s.io/kms/|staging/src/k8s.io/apiserver/pkg/server/options/encryptionconfig/|test/e2e/testing-manifests/auth/encrypt/
+    - name: pull-kubernetes-e2e-kind-nftables
+      optional: true
+      run_if_changed: ^(test/e2e/network/|pkg/apis/networking|pkg/proxy/nftables/)
+    - name: pull-kubernetes-e2e-storage-kind-disruptive
+      optional: true
+      run_if_changed: ^(pkg\/controller\/volume|pkg\/kubelet\/volumemanager|pkg\/volume|pkg\/util\/mount|test\/e2e\/storage|test\/e2e\/testing-manifests\/storage-csi)
+    - name: pull-kubernetes-kind-dra
+      optional: true
+      run_if_changed: /(dra|dynamicresources|resourceclaim|deviceclass|resourceslice|resourceclaimtemplate|dynamic-resource-allocation|pkg/apis/resource|api/resource)/.*.go
+    - name: pull-kubernetes-kind-dra-all
+      optional: true
+      run_if_changed: /(dra|dynamicresources|resourceclaim|deviceclass|resourceslice|resourceclaimtemplate|dynamic-resource-allocation|pkg/apis/resource|api/resource)/.*.go
+    - name: pull-kubernetes-kind-json-logging
+      optional: true
+      run_if_changed: /staging/src/k8s.io/component-base/logs/
+    - name: pull-kubernetes-kind-text-logging
+      optional: true
+      run_if_changed: /staging/src/k8s.io/component-base/logs/
```

Comment:
These two may qualify for merge-blocking now. I'll check...

Having them show up in this list is a useful reminder :sweat_smile:

Reply:
Apropos "merge-blocking": as discussed on Slack, we don't want jobs to run only occasionally *and* be merge-blocking. We currently have one such job. I demoted it to `optional: true`, changed the test so that it errors when it encounters such a job (that's how I found it), and removed the `optional` from the file because all `run_if_changed` jobs now will be optional.